compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/transform-fire/deep-scope.expect.md

```diff
@@ -30,6 +30,7 @@ function Component(props) {
 ## Code
 
 ```javascript
+import { useFire } from "react";
```

Comment:
hmm i think we might want to export this from the compiler runtime? people shouldn't be calling it directly right?

Reply:
Yea, Rick and I are still working that out. Will leave this for now until Rick and I settle on it and will put up a PR to change it assuming we stick with that decision